src/Datepicker.vue

```diff
@@ -310,6 +315,11 @@ export default {
       this.$emit('input', new Date(timestamp))
     },
 
+    clearDate () {
+      this.selectedDate = ''
+      this.$emit('selected', this.selectedDate)
```

Comment:
Glad you liked it! :D im needing this "clear" functionality too, not with a button but i think i can use ref and call your clearDate method :D once merged of course